### ISSUE TITLE
rush search not searching package names

### ISSUE THREAD

**Issue by pcrockett** [DannyBen/rush#81]
As of 0.7.14 it seems Rush doesn't search package names.

Steps to reproduce:

```bash
export RUSH_CONFIG="${HOME}/.config/rush/config"
export RUSH_ROOT="${HOME}/.local/share/rush"
rush clone pcrockett --ssh --default
rush search tagref
```

output of last command is:

```plaintext
Matching packages:


Matching info files:


```

... but _should_ show https://github.com/pcrockett/rush-repo/blob/main/tagref/info

Running Archlinux. I haven't dug into the code yet to troubleshoot, but was wondering first if it broke for anyone else.

**Comment by DannyBen** [comment]
Weird. Search works for my repo without a problem, but I can confirm it does not when I use these repro steps in a clean docker environment:

```
docker run --rm -it dannyben/rush
mkdir -p "${HOME}/.config/rush"
export RUSH_CONFIG="${HOME}/.config/rush/config"
export RUSH_ROOT="${HOME}/.local/share/rush"
rush clone pcrockett --default
rush search zed
```

output:

```
$ rush search zed
Matching packages:


Matching info files:

kbgrep: A search tool optimized for knowledgebases
```

The search for words in package files works, but the first search for package names does not.
I am checking why.

**Comment by DannyBen** [comment]
Found the problem:

The package search `find` is running this:

```
find "$repo_path" -type d -not -path '*/\.*'
```

which ignores any folder that has a `/.` in it (to ignore .git for example).
Since you are storing your repos in `~/.local` - all are ignored during that search.

**Comment by DannyBen** [comment]
Can you try the rush executable from #82 ?

**Comment by pcrockett** [comment]
Fixes my issue, thanks. Though I left a comment on the PR for you.

**Comment by DannyBen** [comment]
Updated the code with:

1. bold instead of blue titles
2. titles will now show the repository name
3. merged multiple `sed` commands to one
4. revised logic for finding package name

https://github.com/DannyBen/rush/blob/79d850b1e391c93ffa4b349eccca8f4ad20ae779/src/search_command.sh#L12-L24

Originally I used `-printf "%h\n" option in `find`, but it seems to be incompatible with some `find` versions.
